tests/acceptance/features/webUITextEditor/editTextFiles.feature

```diff
@@ -22,3 +22,20 @@ Feature: textFiles
     And the user closes the text editor
     And the user opens file "New text file.txt" using the webUI
     Then line 1 of the text should be "other text before stuff"
+
+  @issue-core-36233
+  Scenario: Delete and restore hidden text file
+    Given the user has created a text file with the name "abc.txt"
+    And the user has input "This is a hidden file" in the text area
+    And the user has closed the text editor
+    When the user enables the setting to view hidden files on the webUI
+    And the user renames file "abc.txt" to ".abc.txt" using the webUI
```

Comment:
Does the original problem depend on first doing all this in the UI?
Probably you can just use API `Given` steps to directly create `.abc.txt` with some content, and then delete and restore it with API steps.
In the UI you can just do the editing, and there is a single `Then` API step that checks the downloaded content.